setup.py

```diff
@@ -160,7 +161,7 @@ def get_libs(iface='daal'):
         onedal_lib = [f'onedal_dll{major_version}']
         onedal_dpc_lib = [f'onedal_dpc_dll{major_version}']
     else:
-        libraries_plat = ['onedal_core', 'onedal_thread']
+        libraries_plat = ['onedal_core', 'onedal_thread', 'sycl']
```

Comment:
Just initial workaround against `LinuxDPCPP` check fail. Will be reverted and updated for all jobs

Reply:
this force regular lnx and mac build to fail - you don't need sycl there.